src/components/vuestic-components/va-input/VaInput.vue

```diff
@@ -186,7 +186,7 @@ export default {
 
       return { color: this.$themes.gray }
     },
-    labelStyles () {
+    labelStyle () {
```

Comment:
Not sure about this change. There is inconsistency between develop branch and v0.2.

In develop this property is called labelStyles, but in v0.2 it's labelStyle